CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/http/apis/OpenAi.cs

```diff
@@ -268,7 +402,56 @@ public static class ChatGptExtensions {
         }
 
         public static T ParseNewLineContentAsJson<T>(this ChatGpt.Line newLine) {
-            var responseText = newLine.content;
+            var responseText = (string)newLine.content;
+            if (responseText.StartsWith("```json\n")) {
+                responseText = responseText.Replace("```json\n", "");
+            }
+            if (responseText.EndsWith("\n```")) {
+                responseText = responseText.Replace("\n```", "");
+            }
+            return JsonReader.GetReader().Read<T>(responseText);
+        }
+
+    }
+
+    public static class VisionGptExtention {
+
+        public static void AddImageURL(this ICollection<VisionGpt.Line> self, string url) {
+            var content = new List<Dictionary<string, object>>();
+            content.Add(new Dictionary<string, object>()
+            {
+                {"type","text"},
+                {"text","What's in this image?"},
```

Comment:
This is just one possible phrase, can you make this a parameter of the method. What happens with your test if you dont include that line at all?

Reply:
You are right. This is superfluous. aa31ce0223df19c8d1bc42dd299eed32086211a2